### DISCUSSION TITLE
Load CSS stylesheets of new page

### DISCUSSION THREAD

**Discussion by AaykeN** [swup/swup#519] [Help] [answered]
Hi, Swup fade animation is working fine between pages. However, CSS doesn't work on whichever page I click on. For example, say If I used Grid/Flex on the page that I clicked on, the Grid/Flexbox doesn't work. This only happen when Swup is in use. 

I feel like this has to do with page reload. I tried reading the documentation but still couldn't really solve it.

**Comment by daun** [comment, marked as answer]
**Update**: the [Head Plugin](https://swup.js.org/plugins/head-plugin/) now supports waiting for newly added stylesheets to load before showing the new page.

Previous answer: If you include separate CSS bundles for different subpages, that's to be expected since swup by default won't include stylesheets of the newly loaded page. You can try to bundle the whole site's stylesheet into a single file, that should solve your issue.
